wallet/wallet-cli-lib/src/commands/mod.rs

```diff
@@ -65,6 +65,9 @@ pub enum ColdWalletCommand {
         wallet_path: PathBuf,
         /// The existing password, if the wallet is encrypted.
         encryption_password: Option<String>,
+        /// Force change the wallet type from hot to cold or from cold to hot
+        #[arg(long)]
+        force_change_wallet_type: bool,
     },
```

Comment:
This parameter should be optional, and it should be false by default.

Reply:
clap treats it as a flag so it is false by default and only set to true when you pass --force-change-wallet-type

But can't make it optional in the RPC interface, as even as optional it needs None/null to be passed